RadixWallet/MIGRATE_TO_SARGON/Stage1/Stage1MigrateToSargon+Account.swift

```diff
@@ -42,15 +42,19 @@ extension Account {
 	}
 
 	mutating func hide() {
-		flags.append(.deletedByUser)
+		flags.append(.hiddenByUser)
 	}
 
 	mutating func unhide() {
-		entityFlags.remove(.deletedByUser)
+		entityFlags.remove(.hiddenByUser)
```

Comment:
Notice folder path :) `MIGRATE_TO_SARGON` so perhaps this can be removed altogether and moved to Swift Sargon and probably using Sargon methods/functions.

Would be great with shared functionality with Android Host. I.e. should we simply remove `hiddenByUser` flag? What if one calls this method on an account has been deleted - should not be possibile in UI - but should we throw? Crash? Or do nothing? Would be great to have made that decision once - in Sargon - and  reuse for hosts.

Reply:
There is mark_as_hidden in Sargon which could be used, but I think needs to be done separately and not part of this PR, when we will start migrating more logic from the Wallet to Sargon.